### PR TITLE
Fixing example - attaching additional props using .attr

### PR DESCRIPTION
**Pull request by kelsonpw**
Fixes the example on (https://www.styled-components.com/docs/basics#attaching-additional-props) #460 

Before:
image

After:
image

### CODE REVIEW

**Comment by mxstbr** [comment]
The `margin: ${props => props.margin};` and `padding: ${props => props.padding}` should also both be `props => props.size`, right?

Otherwise this LGTM!

**Comment by kelsonpw** [comment]
@mxstbr Yep good call.  It was a little confusing previously.  Updating both to `props => props.size`.

Let me know if anything else needs changing, but I think its good to go :)

File: sections/basics/attaching-additional-props.md [hunk 1]
```diff
@@ -7,27 +7,26 @@ This way you can for example attach static props to an element, or pass a third-
 Here we render an `Input` component and attach some dynamic and static attributes to it:
 
 ```react
-const Input = styled.input.attrs(({ size }) => ({
+const Input = styled.input.attrs({
```
Comment (probablyup) [inline sections/basics/attaching-additional-props.md]: This object-form attrs is actually deprecated, it'd be good to update it to function version instead:

```js
const Input = styled.input.attrs(props => ({ })`...`
```

Comment (kelsonpw) [reply sections/basics/attaching-additional-props.md]: @probablyup 
I originally tried to write it that way, but the current version doesn't call `.attrs` as a function, it appears.

This site is currently using `"styled-components": "^4.0.0-beta.11", and from xmy testing it isnt until 4.2.0 where the function form works properly, but that breaks the app (.attrs is used in many places.)

The fix would be to update to 4.3.1 and update any `.attrs` app wide to use the function form right?  I would be happy to do the work and submit a PR.

**Pull request merged**